mackup/applications/starship.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = Starship
+
+[configuration_files]
+.config/starship.toml
```

Comment:
I believe this should use the [built-in XDG support](https://github.com/lra/mackup/tree/master/doc#add-support-for-an-application-using-the-xdg-directory) instead: 
```suggestion
[xdg_configuration_files]
starship.toml
```

Reply:
seems starship is not using XDG: https://github.com/starship/starship/issues/896

Reply:
Oh, indeed. I just assumed it was since the file is in `.config`. Sorry about that!

Reply:
Yeah it's super confusing.